templates/facets/facets-item-list.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Override facets item lists, alter title, inject classes.
+ */
+#}
+<div class="facets-widget- {{- facet.widget.type -}} ">
+  {% if facet.widget.type -%}
+    {%- set attributes = attributes.addClass('item-list__' ~ facet.widget.type) %}
+  {%- endif %}
+  {% if facet.widget.type == 'dropdown' -%}
+    {%- set attributes = attributes.addClass('form-select') %}
+  {%- endif %}
+  {% if facet.widget.type == 'checkbox' -%}
+    {%- set attributes = attributes.addClass('oel-facets-checkbox-list') %}
+  {%- endif %}
+  {% if items or empty %}
+    {%- if title is not empty -%}
+      <legend class="col-form-label"><span class="fieldset-legend">{{ title }}</span></legend>
+    {%- endif -%}
+
+    {%- if items -%}
+      <{{ list_type }}{{ attributes }}>
+        {%- for item in items -%}
+          <li{{ item.attributes.addClass('mb-2') }}>{{ item.value }}</li>
```

Comment:
I lowered the margin here from mb-3 like in netlify to mb-2, also created a ticket to follow up